packages/volto-social-blocks/src/components/Blocks/Flickr/Data.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { BlockDataForm } from '@plone/volto/components';
+import { isValidFlickrId } from '../../../helpers';
+import { flickrSchema } from './schema';
+import { useIntl } from 'react-intl';
+
+const FlickrBlockData = (props) => {
+  const { data, block, onChangeBlock, blocksConfig, navRoot, contentType } =
+    props;
+  const intl = useIntl();
+  const schema = flickrSchema({ ...props, intl });
+  Object.keys(schema.properties).forEach((key) => {
```

Comment:
I fixed the defaults, now they should work in the `schema.js` try it out, but leaving this here does not hurt...